scripts/privacyidea

```diff
@@ -520,7 +524,7 @@ def nitrokey_mass_enroll(args, client):
     NK.admin_login(password)
     while True:
         print("Please insert the next Nitrokey!")
-        raw_input("Press [ENTER] when ready.")
+        input("Press [ENTER] when ready.")
```

Comment:
I think it would be good if we do ``from six.moves import input``, in order not to use the unsafe ``input`` in Python 2 :)

Reply:
Will do.

Reply:
fixed in 13670ac